### DISCUSSION TITLE
Option to rotate Track Map

### DISCUSSION THREAD

**Discussion by RCD800** [f1multiviewer/issue-tracker#204] [Feature requests] [answered]
As said in the title
How it looked during Monza FP1:
![image](https://user-images.githubusercontent.com/85562954/189358180-f2f4806d-f87d-4387-9070-24f0d77b15fa.png)
Being that tall makes it hard to see names unless the window is bigger which may be a problem when there's limited space for the track map, discord, and the feed itself.
Here's how F1TV on screen map looked during Monza FP1:
![image](https://user-images.githubusercontent.com/85562954/189358444-d20fc6cf-1038-419b-b4a0-937a42af3655.png)

**Comment by f1multiviewer** [comment, marked as answer]
I've added track rotations (and the track rotation used by F1's driver tracker) for each track, so this will no longer be an issue going forward!
